system/nxplayer/nxplayer.c

```diff
@@ -314,7 +314,7 @@ static int nxplayer_opendevice(FAR struct nxplayer_s *pplayer, int format,
 
 #ifdef CONFIG_NXPLAYER_INCLUDE_DEVICE_SEARCH
     {
-      struct audio_caps_s caps;
+      struct audio_caps_s cap;
```

Comment:
what is the warning reposted here?

Reply:
checkpatch.sh failed on Line:398,  the reason is "found Long Line".  So change caps to cap.

Reply:
Ok. Tricky way to handle long lines :), but still it works. Usually people do
```
for (x = 0;
     x < sizeof(caps.ac_controls);
     x++)
```